NodeJS/src/Utils.ts

```diff
@@ -15,6 +15,7 @@ const logger = log4js.getLogger('Utils')
 const URL = require('url').URL
 // Username in mpact.user and ac_station.user is 50 chars long; Bulk upload uses 49, so does here.
 const MAX_USERNAME_LENGTH = 49
+const MAX_FIRST_LAST_NAME_LENGTH = 16
```

Comment:
only 16 characters for both a first and last name? that's too short, way too short, this should be _at least_ 32, and I'd vote for 64.

Reply:
nvm, misunderstood

Reply:
each part (first name and last name) could be up to 16 chars. so, total length up to 32 chars.
sorry for confusing const and validator names. I followed a front-end validation, which treats first and last names similarly.
and I didn't name the validator and const as "...firstOrLast..." because it could include other parts such as middle name, nickname, title, number, etc, so 'OR' could also mislead.